src/merge.js

```diff
@@ -51,18 +50,18 @@ class Merge {
     let parentId = path.dirname(doc._id)
     if (parentId === '.') { return }
 
-    let folder
     try {
-      folder = await this.pouch.db.get(parentId)
-    } catch (_) {}
-    if (folder) { return }
+      let folder = await this.pouch.db.get(parentId)
+      if (folder) { return }
+    } catch (err) {
+      if (err.status !== 404) { log.warn(err) }
```

Comment:
Oh my... I believe the `catch (_) {}` was a direct translation of the old callback-based implementation, which I didn't fix. Thanks.